src/nunit.analyzers/SourceCommon/SourceHelpers.cs

```diff
@@ -79,12 +79,27 @@ internal static class SourceHelpers
                 return null;
             }
 
-            var symbol = attributeInformation.SourceType
-                .GetMembers(attributeInformation.SourceName)
+            return attributeInformation.SourceType.GetMember(attributeInformation.SourceName);
+        }
+
+        public static ISymbol? GetMember(this INamedTypeSymbol typeSymbol, string name)
+        {
+            ISymbol? symbol = typeSymbol
+                .GetMembers(name)
                 .FirstOrDefault(m => m.Kind == SymbolKind.Field
                     || m.Kind == SymbolKind.Property
                     || m.Kind == SymbolKind.Method);
 
+            if (symbol is null && typeSymbol.BaseType != null)
```

Comment:
I think there needs to be a loop. A test that places the test case source in the base class of the base class would indicate this.

Reply:
Never mind, I missed the recursion.